Editor/Editors/CinemachineVirtualCameraBaseEditor.cs

```diff
@@ -648,5 +684,38 @@ void DrawFocalLengthControl(Rect rect, SerializedProperty property, GUIContent l
 #endif
         }
     }
+    
+#if CINEMACHINE_UNITY_INPUTSYSTEM
+    /// <summary>
+    /// Provides a simple API to get a default XY input action from the InputSystem package.
+    /// </summary>
+    class CinemachineDefaultMouseInput
```

Comment:
This feels complicated for nothing.  Why not just a local static InputActionReference that gets initialized if it's null?